README.md

```diff
@@ -1,24 +1,84 @@
-# README
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+# Budget-App
+This is a mobile web application where you can manage your budget: you have a list of transactions associated with a category, so that you can see how much money you spent and on what.
 
-Things you may want to cover:
+## Built With
+- Languages used
+  - Ruby
+  - HTML
+  - CSS
+  - Rails
+  
+- Technologies used
+  - PostgreSql
+  - Rspec
+  - Capybara
 
-* Ruby version
 
-* System dependencies
+## Getting Started
 
-* Configuration
+### Prerequisite
+Go ahead and check the most recent version of each of these:
+- Ruby (ruby --version)
+- PostgreSQL (PostgreSQL --version)
+- Node.js (node --version)
+- Yarn (yarn --version)
+- Rails (rails --version)
 
-* Database creation
+## Clone this repository
 
-* Database initialization
+To get a local copy up and running follow these simple example steps.
 
-* How to run the test suite
+To clone the repo run:
+```
+git clone https://github.com/kelomo2502/budget-app.git
+```
+To access the cloned directory, run:
+```
+cd budget-app
+```
+Install the required dependecies with:
+```
+bundle install
+```
+Install npm dependencies with:
+```
+npm install
+```
+Before running below commands update config/database.yml with your local DBcredentials. Then run:
+```
+rake db:create
+```
+```
+rake db:migrate
+```
+## Authors
 
-* Services (job queues, cache servers, search engines, etc.)
+👤 **Oyewunmi Gbenga**
 
-* Deployment instructions
+- GitHub: [@githubhandle](https://github.com/kelomo2502)
+- Twitter: [@twitterhandle](https://twitter.com/kelomoJs)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/oyewunmi-gbenga)
 
-* ...
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/kelomo2502/budget-app/issues/1).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
```

Comment:
-  You should render an "Acknowledgement" or give appropriate credit to [Gregoire Vella on Behance](https://www.behance.net/gregoirevella) the [author of the original design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), as required by the [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/). It is a professional thing to do when using the resources of other developers in your work.